### PR TITLE
Added Headers and footers, PPTX to image, PPTX to PDF samples

### PR DESCRIPTION
**Pull request by DharanitharanA**
Hi all,
With this PR, we have added GitHub samples for below topics:

- Headers and footers
- PPTX to image
- PPTX to PDF

Regards,
Dharanitharan A

### CODE REVIEW

File: PPTX-to-PDF-conversion/Set-PDF-conformance-level/.NET-Framework/Set-PDF-conformance-level/Program.cs [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        {
+            //Load or open an PowerPoint Presentation.
+            using (IPresentation pptxDoc = Presentation.Open(@"../../Data/Template.pptx"))
+			{
+                //Initialize the conversion settings
+                PresentationToPdfConverterSettings pdfConverterSettings = new PresentationToPdfConverterSettings();
+                //Set the Pdf conformance level to A1B
+                pdfConverterSettings.PdfConformanceLevel = PdfConformanceLevel.Pdf_A1B;
+                //Convert the documents by passing the settings as parameter.
+                PdfDocument pdfDoc = PresentationToPdfConverter.Convert(pptxDoc, pdfConverterSettings);
```
Comment (Ramaraj-Marimuthu) [inline PPTX-to-PDF-conversion/Set-PDF-conformance-level/.NET-Framework/Set-PDF-conformance-level/Program.cs:24]: @DharanitharanA - Please dispose the `PdfDocument` properly

**Review by Ramaraj-Marimuthu — CHANGES_REQUESTED** [review]

File: PPTX-to-Image-conversion/Change-resolution-of-converted-image/.NET-Framework/Change-resolution-of-converted-image/Program.cs [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+                //Create a bitmap of specific width and height.
+                Bitmap bitmap = new Bitmap(customWidth, customHeight, PixelFormat.Format32bppPArgb);
+                //Get the graphics from image.
+                Graphics graphics = Graphics.FromImage(bitmap);
+                //Set the resolution.
+                bitmap.SetResolution(graphics.DpiX, graphics.DpiY);
+                //Recreate the image in custom size.
+                graphics.DrawImage(System.Drawing.Image.FromStream(stream), new Rectangle(0, 0, bitmap.Width, bitmap.Height));
+                //Save the image as bitmap .
+                bitmap.Save("../../ImageOutput" + Guid.NewGuid().ToString() + ".jpeg");
```
[the remark this answers is not in the document]
Comment (DharanitharanA) [reply PPTX-to-Image-conversion/Change-resolution-of-converted-image/.NET-Framework/Change-resolution-of-converted-image/Program.cs]: Disposed the bitmap properly.

File: PPTX-to-Image-conversion/Convert-PowerPoint-presentation-to-Image/.NET-Framework/Convert-PowerPoint-presentation-to-Image/Program.cs [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+                {
+                    //Set the scaling mode as best.
+                    ScalingMode = Syncfusion.OfficeChart.ScalingMode.Best
+                };
+                //Convert entire Presentation to images.
+                Image[] images = pptxDoc.RenderAsImages(Syncfusion.Drawing.ImageType.Metafile);
+                //Save the image to file system.
+                foreach (Image image in images)
+                {
+                    image.Save("../../ImageOutput" + Guid.NewGuid().ToString() + ".png");
```
[the remark this answers is not in the document]
Comment (DharanitharanA) [reply PPTX-to-Image-conversion/Convert-PowerPoint-presentation-to-Image/.NET-Framework/Convert-PowerPoint-presentation-to-Image/Program.cs]: Disposed the image properly.

File: PPTX-to-PDF-conversion/Apply-substitution-font-name/.NET/Apply-substitution-font-name/Program.cs [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+//Load or open an PowerPoint Presentation.
+using FileStream inputStream = new(Path.GetFullPath(@"../../../Data/Template.pptx"), FileMode.Open, FileAccess.Read, FileShare.ReadWrite);
+//Open an existing PowerPoint presentation.
+using IPresentation pptxDoc = Presentation.Open(inputStream);
+// Initialize the 'SubstituteFont' event to set the replacement font.
+pptxDoc.FontSettings.SubstituteFont += SubstituteFont;
+//Convert the PowerPoint presentation to PDF file.
+using PdfDocument pdfDocument = PresentationToPdfConverter.Convert(pptxDoc);
+//Create new instance of file stream.
+FileStream pdfStream = new("../../../PPTXToPDF.pdf", FileMode.Create);
```
[the remark this answers is not in the document]
Comment (DharanitharanA) [reply PPTX-to-PDF-conversion/Apply-substitution-font-name/.NET/Apply-substitution-font-name/Program.cs]: Added FileStream under using scope.

File: PPTX-to-PDF-conversion/Convert-PowerPoint-into-accessible-PDF/.NET/Convert-PowerPoint-into-accessible-PDF/Convert-PowerPoint-into-accessible-PDF.csproj [hunk 5]
```diff
@@ -0,0 +1,16 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net6.0</TargetFramework>
+    <RootNamespace>Convert_PowerPoint_presentation_to_PDF</RootNamespace>
```
[the remark this answers is not in the document]
Comment (DharanitharanA) [reply PPTX-to-PDF-conversion/Convert-PowerPoint-into-accessible-PDF/.NET/Convert-PowerPoint-into-accessible-PDF/Convert-PowerPoint-into-accessible-PDF.csproj]: Modified the namespace name properly

File: PPTX-to-PDF-conversion/Set-PDF-conformance-level/.NET-Framework/Set-PDF-conformance-level/Program.cs (hunk 1 above)
Comment (DharanitharanA) [reply PPTX-to-PDF-conversion/Set-PDF-conformance-level/.NET-Framework/Set-PDF-conformance-level/Program.cs]: Disposed the PdfDocument properly

**Review by Ramaraj-Marimuthu — APPROVED** [review]
I don't have any concerns in this code changes.

**Pull request merged**